### PR TITLE
Changed the background color of welcome email handle in mailer.scss

### LINKED ISSUE #27707: Welcome email handle color is black the background color is also black so its not legible
### Steps to reproduce the problem

1.Load [http://localhost:3000/rails/mailers/user_mailer/welcome](url) in development environment
2.
3.
...

### Expected behaviour

Should be able to see the handle

### Actual behaviour

The dark text isnt legible

### Detailed description

Screenshot 2023-11-05 at 11 22 19 AM

### Mastodon instance

mastodon.social

### Mastodon version

v4.1.2

### Browser name and version

Firefox 105.0.3

### Operating system

macOS 13.4.1

### Technical details

_No response_

### PR DESCRIPTION
**Pull request by Elgueromiranda**
The handle isnt legible because the text is black and the background is dark

### CODE REVIEW

**Comment by Elgueromiranda** [comment]
[https://github.com/mastodon/mastodon/issues/27707](url)

**Comment by Elgueromiranda** [comment]
Closes #6938

**Comment by Elgueromiranda** [comment]
fixes #6938

**Comment by rugk** [comment]
Awesome!

Just FYI, cause you tried it: You can (automatically) [let issues close](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) when a PR is merged by adding some "magic" text to your PR _body_. Or manually link it at the right. Using the codeword in a comment does not help.

File: app/javascript/mastodon/features/emoji/unicode_to_filename.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 // taken from:
 // https://github.com/twitter/twemoji/blob/47732c7/twemoji-generator.js#L848-L866
-exports.unicodeToFilename = (str) => {
+export const unicodeToFilename = (str) => {
```
Comment (renchap) [inline app/javascript/mastodon/features/emoji/unicode_to_filename.js:6]: This change is unrelated, please remove your first commit from your PR. The issue you are trying to fix here has been fixed in `main`, so a rebase should fix this.

File: app/javascript/mastodon/features/emoji/unicode_to_unified_name.js [hunk 2]
```diff
@@ -8,8 +8,7 @@ function padLeft(str, num) {
 
   return str;
 }
-
-exports.unicodeToUnifiedName = (str) => {
+export const unicodeToUnifiedName = (str) => {
```
Comment (renchap) [inline app/javascript/mastodon/features/emoji/unicode_to_unified_name.js:11]: Same here

**Review by renchap — CHANGES_REQUESTED** [review]

**Comment by Gargron** [comment]
We are working on an e-mail redesign.

**Pull request closed without merging**